Data/ByteString/Char8.hs

```diff
@@ -837,6 +839,29 @@ zip ps qs
 zipWith :: (Char -> Char -> a) -> ByteString -> ByteString -> [a]
 zipWith f = B.zipWith ((. w2c) . f . w2c)
 
+--
+-- | A specialised version of `zipWith` for the common case of a
+-- simultaneous map over two ByteStrings, to build a 3rd.
+packZipWith :: (Char -> Char -> Char) -> ByteString -> ByteString -> ByteString
```

Comment:
Could it be expressed via `Data.ByteString.packZipWith`, using `c2w` and `w2c`?

Reply:
Yes it could. Done in 6b454b2 and e0946d1